### DISCUSSION TITLE
How to install system informer

### DISCUSSION THREAD

**Discussion by MonsterSe7en** [winsiderss/systeminformer#1280] [Questions / Answers]
![image](https://user-images.githubusercontent.com/21208241/174305685-7d99dcc6-06dd-4b18-bab1-fc2bcdc1ad8c.png)

[Project Website](https://systeminformer.sourceforge.io/) - [Project Downloads](https://systeminformer.sourceforge.io/downloads.php)

The above links don't seem to work. How do I install the application, I've been a fan of process hacker!

Project websites takes me to a blank page; and

Project Downloads gives me the following error
![image](https://user-images.githubusercontent.com/21208241/174306074-5be38300-9adc-4442-acea-754943fb989d.png)

**Comment by dmex** [comment]
As mentioned in the announcement... we're still migrating.

**Reply by MonsterSe7en** [reply]
Thank You. Looking forward to the release

Cheers...

**Reply by TPS** [reply]
To followup, https://processhacker.sourceforge.io/nightly.php _was_ a reliable link to @ least _try_ whatever nightly might've compiled for PH.  Is there yet [an equivalent URL](https://systeminformer.sourceforge.io/nightly.php) for SI?

**Reply by jxy-s** [reply]
> Is there yet an equivalent URL for SI?

No, we haven't done a build of build of SI nightly yet.

**Reply by earthsound** [reply]
> To followup, https://processhacker.sourceforge.io/nightly.php _was_ a reliable link to @ least _try_ whatever nightly might've compiled for PH. Is there yet [an equivalent URL](https://systeminformer.sourceforge.io/nightly.php) for SI?

Nightlies are now available.

**Reply by TPS** [reply]
This is indeed excellent news.  Can't wait to [try it](https://systeminformer.sourceforge.io/canary)!
